### ISSUE TITLE
Option to refresh shop items

### ISSUE THREAD

**Issue by joewis** [diasurgical/devilutionX#1109]
Griswold has a new set of items on sale whenever starting a new multiplayer game.

Instead of repeatedly restarting games until he has something useful, we might as well have an option to refresh his shop. Mindlessly browsing through Griswold's would also save me money compared to doing that at Amazon.

It feels a bit like cheating, but can already be achieved by starting new games, just more convenient.

**Comment by Chance4us** [comment]
The same would make sense for Adria and Pepin as well.

**Comment by joewis** [comment]
I'm a bit critical with Adria as she's selling elixirs. If you can just equip your character with gold and keep refreshing and chugging elixirs, it changes the gameplay a bit. (But tbh, I've been doing that the slow way and still have fun.)
It may be better to introduce a menu for witch premium items and only refresh those.

**Comment by galaxyhaxz** [comment]
Better solution is to remove elixirs from stores. I think they should be more rare given how easy it is already to abuse them. Even without refresh you can still make new games for Pepin trips and buy out elixirs so long as you have gold. Having to find them makes it more challenging.

**Comment by joewis** [comment]
I don't disagree, but that would be a bit drastic...

**Comment by galaxyhaxz** [comment]
Yeah it'd be more for mod makers. On that note the same could be said for spell books, where again gold can buy you maximum spells. Removing them makes it more fun and challenging.

**Comment by julealgon** [comment]
I really don't like this proposal, but it is hard to argue with the "you can already do this in a more tedious way" line of thinking...

I wish we could just disallow the items from refreshing at all, but even that is hard to do since we don't have much control of when "new game" is selected 😕 

I'm thorn on this one... it completely and utterly defeats the purpose of a list in Griswold.

The only way I'd have a chance to support it is if we check whether the player has entered the dungeon at least once or not, and only allow this refresh if he hasn't on that game. Then it won't feel so easy to exploit the vendor on standard games where you don't want to lose your progress.
